i18n/locales/es.ts

```diff
@@ -0,0 +1,376 @@
+
+      feedback: {
+        title: 'Comentarios',
+        paragraph_one:
+          'Una buena forma de contribuir es repasar el capítulo 1 y darnos tu opinión sobre la experiencia general utilizando este formulario.',
+      },
+
+      contribute: {
+        title: 'Contribuir',
+        paragraph_one:
+          'Todas las sugerencias son bienvenidas, incluidos los cambios de contenido, mecánica de juego - realmente cualquier cosa. También estamos siempre buscando ayuda para revisar y <a href="https://leaf-singer-0fc.notion.site/How-to-QA-5177e63f65a4406da01bc57d886b5ac2" target="_blank" rel="noreferrer">Controlar la calidad de los cambios de código</a>. Para mejorar el código, puedes abrir directamente una incidencia o enviar un pull request en GitHub.',
+        paragraph_two:
+          'Si quieres participar en el diseño, la historia o cualquier otra cosa, únete a nosotros en el canal #saving-satoshi en el slack <a href="https://bitcoin.design/contribute/" target="_blank" rel="noreferrer">Bitcoin Design</a>. Menciona cómo te gustaría contribuir y te ayudaremos a orientarte en la dirección correcta.',
+      },
+    },
+
+    contributors: {
+      title: 'Disfrutalo!',
+      contributions_by: 'Contribuciones de',
+      many_more:
+        ', <href="https://VEINTIUNO.world/libro/salvando-satoshi/" target="_blank">@VEINTIUNO.world</a> y <a href="https://github.com/saving-satoshi/saving-satoshi/graphs/contributors" target="_blank" rel="noreferrer">muchos más</a>.',
+    },
+
+    satoshi_needs_you: 'Ahora date prisa, Satoshi te necesita.',
+  },
+
+  ///CHAPTERS TITLE PAGE
+  chapter_one: {
+    title: 'Secretos a la vista de todos',
+    paragraph_one:
+      'El año es 2139. Faltan dos semanas para que se extraiga el último bitcoin. Desde hace meses, un reloj hace tictac en la Plaza Satoshi. El mundo espera el último bloque. De repente, la red se detiene.',
+
+    ///CHAPTER 1
+    intro_one: {
+      title: 'Génesis',
+      paragraph_one: 'Unos instantes después, tu Pantalla se activa.',
+      paragraph_two: '-Deborah Chunk: "Thomas Vanderpoole. Como CEO de BitRey, diriges uno de los mayores pools de minería de bitcoins del mundo. También fabrica máquinas de minería bitcoin. ¿Qué está ocurriendo? ¿Está muriendo bitcoin?”',
+      paragraph_three: '-Vanderpoole: "Sí, Deborah. Como mi padre y su padre antes que él. Los Vanderpoole han estado minando desde el Bloque 21.000. Por eso puedo decir con seguridad que los mineros de todo el mundo están causando estos retrasos al apagar sus máquinas. Esto es una protesta. Nadie quiere que bitcoin deje de emitirse a los 21 millones. No podemos sobrevivir sólo con las comisiones.”',
+      start: 'Continúa',
+    },
+
+    intro_two: {
+      title: 'Génesis',
+      paragraph_one:
+        'Recibes un holocat de alguien que usa el nombre de Satoshi Nakamoto. (Es como cualquier otro e-holograma, pero este tiene forma de gato.) Abres el holocat abucheando su nariz.',
```

Comment:
I did not even know what a "boop" is, but according to [this reddit post](https://www.reddit.com/r/learnspanish/comments/mlbi0q/boop/) there is no translation for that. There they suggest using the same "boop" (or its Spanish adaptation "bup"), but I have never in my life heard anyone call that a "boop" in Spanish. I think putting it in terms of "golpear" (hit) would keep the meaning closer to the original (technically "dándole un golpecito en" would be better, but that would make the text too long). "Abuchear" is to boo, so it makes no sense in this context:

```suggestion
        'Recibes un holocat de alguien que usa el nombre de Satoshi Nakamoto. (Es como cualquier otro e-holograma, pero este tiene forma de gato.) Abres el holocat golpeando su nariz.',
```